src/e-pr.js

```diff
@@ -188,5 +195,231 @@ program
     }
 
     return open(`${repoBaseUrl}/compare/${comparePath}?${querystring.stringify(queryParams)}`);
-  })
-  .parse(process.argv);
+  });
+
+program
+  .command('download-dist <pull_request_number>')
+  .description('Download a pull request dist')
+  .option(
+    '--platform [platform]',
+    'Platform to download dist for. Defaults to current platform.',
+    process.platform,
+  )
+  .option(
+    '--arch [arch]',
+    'Architecture to download dist for. Defaults to current arch.',
+    process.arch,
+  )
+  .option(
+    '-o, --output <output_directory>',
+    'Specify the output directory for downloaded artifacts. ' +
+      'Defaults to ~/.electron_build_tools/artifacts/pr_{number}_{platform}_{arch}',
+  )
+  .option(
+    '-s, --skip-confirmation',
+    'Skip the confirmation prompt before downloading the dist.',
+    !!process.env.CI,
+  )
+  .action(async (pullRequestNumber, options) => {
+    if (!pullRequestNumber) {
+      fatal(`Pull request number is required to download a PR`);
+    }
+
+    d('checking auth...');
+    const auth = await getGitHubAuthToken(['repo']);
+    const octokit = new Octokit({ auth });
+
+    d('fetching pr info...');
+    let pullRequest;
+    try {
+      const { data } = await octokit.pulls.get({
+        owner: 'electron',
+        repo: 'electron',
+        pull_number: pullRequestNumber,
+      });
+      pullRequest = data;
+    } catch (error) {
+      console.error(`Failed to get pull request: ${error}`);
+      return;
+    }
+
+    if (!options.skipConfirmation) {
+      const isElectronRepo = pullRequest.head.repo.full_name !== 'electron/electron';
+      const { proceed } = await inquirer.prompt([
+        {
+          type: 'confirm',
+          name: 'proceed',
+          message: `You are about to download artifacts from:
+
+“${pullRequest.title} (#${pullRequest.number})” by ${pullRequest.user.login}
+${pullRequest.head.repo.html_url}${isElectronRepo ? ' (fork)' : ''}
+
+Proceed?`,
+        },
+      ]);
```

Comment:
Personally I think I'd prefer this default to false so user's can't fat finger through, but it's not a sticking point.

Reply:
Reasonable preference, I'll add it 👍